float64 x
float64 y
float64 z
float64 a
float64 b
float64 c
string[] paramValue
---
int32 res